build/BranchInfo.props

```diff
@@ -15,20 +15,21 @@
       Microsoft.ML.LightGbm;
       Microsoft.ML.Mkl.Components;
       Microsoft.ML.Mkl.Redist;
+      Microsoft.ML.TensorFlow;
```

Comment:
Have we done the appropriate work to ensure the APIs we have in this package are the correct long term APIs we want to go with?

Reply:
We are confident with the APIs even with respect to supporting TensorFlow 2.0?